projects/observability/src/shared/components/span-detail/call/headers/span-detail-call-headers.component.ts

```diff
@@ -33,14 +41,24 @@ export class SpanDetailCallHeadersComponent implements OnChanges {
   @Input()
   public title?: string;
 
+  @Input()
+  public filterName: string = '';
+
   public records$?: Observable<ListViewRecord[]>;
 
   public label?: string;
 
+  public constructor(private readonly explorerService: ExplorerService) {}
+
   public ngOnChanges(): void {
     this.buildRecords();
   }
 
+  public getExploreNavigationParams = (record: ListViewRecord): Observable<NavigationParams> =>
+    this.explorerService.buildNavParamsWithFilters(ScopeQueryParam.EndpointTraces, [
+      { field: this.filterName, subpath: record.key, operator: FilterOperator.Equals, value: record.value }
```

Comment:
`filterName` -> What is it representing exactly? attribute name?

Reply:
Yes, attribute name seems clearer?

Reply:
Yeah, or just `field` or `fieldName` is also better.

Reply:
done